src/modules/esl-popup/core/esl-popup.ts

```diff
@@ -331,23 +331,31 @@ export class ESLPopup extends ESLToggleable {
   }
 
   /** Actions to execute on activator scroll event. */
-  @listen({auto: false, event: 'scroll', target: ($popup: ESLPopup) => $popup.scrollTargets})
+  @listen({auto: false, group: 'observer', event: 'scroll', target: ($popup: ESLPopup) => $popup.scrollTargets})
   protected _onActivatorScroll(e: Event): void {
     if (this._updateLoopID) return;
     this._updatePosition();
   }
 
-  @listen({auto: false, event: 'transitionstart', target: document.body})
+  @listen({auto: false, group: 'observer', event: 'transitionstart', target: document.body})
   protected _onTransitionStart(): void {
     this._startUpdateLoop();
   }
 
-  @listen({auto: false, event: 'resize', target: window})
+  @listen({auto: false, group: 'observer', event: 'resize', target: window})
   @decorate(rafDecorator)
   protected _onResize(): void {
     this._updatePosition();
   }
 
+  @listen({auto: false, group: 'observer', event: ($popup: ESLPopup) => $popup.REFRESH_EVENT, target: window})
+  protected _onRefresh(e: Event): void {
+    const {target} = e;
+    if (!isElement(target)) return;
+    const {activator, $container} = this;
+    if ($container === target || this.contains(target) || activator?.contains(target)) this._updatePosition();
```

Comment:
In our case:
```
<div resizable>
  <div class = "something" >
      <esl-trigger>...<esl-trigger>
      <esl-popup-placeholder>...</esl-popup-placeholder>  
   </div>
   <div class="hey not only popup interested in esl:refresh"  
     ...
    </div>
 </div> 
 ```

so as a an `div[resizble]` maintainers we do not care about content we dispatch `esl:refres` on   `div[resizble]`


I'd suggest using our standard check for the third condition
```suggestion
    const {activator, $container} = this;
    if ($container === target || this.contains(target) || isRelativeNode(this.activator, target)) this._updatePosition();
```